File: sources/Blue Zentih 728/club/bluezenith/module/modules/render/Animations.java
package club.bluezenith.module.modules.render;

import club.bluezenith.module.Module;
import club.bluezenith.module.ModuleCategory;
import club.bluezenith.module.value.types.*;

public class Animations extends Module {
    public static Animations module;
    public final ModeValue anim = new ModeValue("Animation", "1.7", true, null, "1.7", "Exhibition", "Bruh", "Spin", "Rotating", "Swank").setIndex(1);
    public final FloatValue itemPosX = new FloatValue( "ItemPosX",0, -100, 100, 1, true, null).setIndex(2);
    public final FloatValue itemPosY = new FloatValue( "ItemPosY",0, -100, 100, 1, true, null).setIndex(3);
    public final FloatValue itemPosZ = new FloatValue( "ItemPosZ",0, -100, 100, 1, true, null).setIndex(4);
    public final FloatValue itemScale = new FloatValue( "ItemScale",1, 0.1f, 5, 0.05F, true, null).setIndex(5);
    public final BooleanValue slowSwing = new BooleanValue("SlowSwing", false, true).setIndex(10);
    public final BooleanValue slowSwingBlock = new BooleanValue("SlowSwingBlock", false, false, slowSwing::get).setIndex(11);
    public final IntegerValue slowSwingValue = new IntegerValue("SlowSwingValue", 50, 0,100,1,true, slowSwing::get).setIndex(12);
    @SuppressWarnings("unused")
    private final ActionValue reset = new ActionValue("Reset",() -> {
        this.anim.set("1.7");
        this.itemPosX.set(0f);
        this.itemPosY.set(0f);
        this.itemPosZ.set(0f);
        this.itemScale.set(1F);
        this.slowSwing.set(false);
        this.slowSwingValue.set(0);
    }).setIndex(13);

    public Animations() {
        super("Animations", ModuleCategory.RENDER, "animations", "anim");
        module = this;
    }

    @Override
    public String getTag() {
        return anim.get();
    }
}

File: sources/Kevin Reborn/kevin/module/modules/render/Animations.kt
/*
 * This program is free software: you can redistribute it and/or modify
 *   it under the terms of the GNU General Public License as published by
 *   the Free Software Foundation, either version 3 of the License, or
 *   (at your option) any later version.
 *
 *   This program is distributed in the hope that it will be useful,
 *   but WITHOUT ANY WARRANTY; without even the implied warranty of
 *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *   GNU General Public License for more details.
 *
 *   You should have received a copy of the GNU General Public License
 *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
package kevin.module.modules.render

import kevin.module.*

class Animations : Module("Animations","Changes animations.", category = ModuleCategory.RENDER) {
    val animations = ListValue(
        "Preset", arrayOf(
            "Akrien", "Avatar", "ETB", "Exhibition", "Kevin", "Push", "Reverse",
            "Shield", "SigmaNew", "SigmaOld", "Slide", "SlideDown", "HSlide", "Swong", "VisionFX",
            "Swank", "Jello", "LiquidBounce","Rotate"
        ),
        "SlideDown"
    )

    var translateX = FloatValue("TranslateX", 0.0f, 0.0f, 1.5f)
    var translateY = FloatValue("TranslateY", 0.0f, 0.0f, 0.5f)
    var translateZ = FloatValue("TranslateZ", 0.0f, 0.0f, -2.0f)
    val itemPosX = FloatValue("ItemPosX", 0.56F, -1.0F, 1.0F)
    val itemPosY = FloatValue("ItemPosY", -0.52F, -1.0F, 1.0F)
    val itemPosZ = FloatValue("ItemPosZ", -0.71999997F, -1.0F, 1.0F)
    var itemScale = FloatValue("ItemScale", 0.4f, 0.0f, 2.0f)
    val onlyOnBlock = BooleanValue("OnlyBlock",false)
    val rotationItem = BooleanValue("RotationItem",false)
    val animationSpeed = IntegerValue("AnimationSpeed",6,1,30)

    override val tag: String
        get() = animations.get()
}